assets/root-files/travis.yml

```diff
@@ -19,17 +21,16 @@ notifications:
   email: false
  
 install:
-  - if [[ $TRAVIS_PHP_VERSION != '5.6' && $TRAVIS_PHP_VERSION != 'hhvm' && $TRAVIS_PHP_VERSION != '7' ]]; then phpenv config-rm xdebug.ini;  fi
+  - if [[ $EXECUTE_TEST_COVERALLS != 'true' ]]; then phpenv config-rm xdebug.ini ; fi
   - composer install --no-interaction --prefer-source
  
 before_script:
-  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then mkdir -p build/logs; fi
+  - if [[ $EXECUTE_TEST_COVERALLS = 'true' ]]; then mkdir -p build/logs ; fi
  
 script:
-  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then php ./vendor/bin/phpunit -c phpunit.xml.travis --coverage-clover build/logs/clover.xml; fi
-  - if [[ $TRAVIS_PHP_VERSION != '5.6' ]]; then php ./vendor/bin/phpunit -c phpunit.xml.travis; fi
- 
-  - if [[ $TRAVIS_PHP_VERSION = '5.6' ]]; then php ./vendor/bin/php-cs-fixer fix -v --diff --dry-run --config-file=.php_cs; fi
+  - if [[ $EXECUTE_TEST_COVERALLS = 'true' ]]; then php ./vendor/bin/phpunit -c phpunit.xml.travis --coverage-clover build/logs/clover.xml ; fi
```

Comment:
And... learned something new today: http://stackoverflow.com/a/3265826/31459 indicates that `==` is a synonym for `=`. I swear I've had bash report errors on this before. :)